src/util.c

```diff
@@ -633,3 +634,20 @@ All facets connected.  No further nearby check necessary.\n");
     stl_verify_neighbors(stl);
   }
 }
+
+void stl_write_error_message(stl_file *stl, const char *msg){
+
+  if(stl->error_buffer != NULL)
+    free(stl->error_buffer);
+
+  // Check for library errors and append additional error message if available.
```

Comment:
The codebase now only uses C89-style comments

```suggestion
  /* Check for library errors and append additional error message if available. */
```